libs/features/mas/web-components/src/hydrate.js

```diff
@@ -237,6 +251,7 @@ export async function hydrate(fragmentData, merchCard) {
     merchCard.removeAttribute('badge-color');
     merchCard.removeAttribute('badge-text');
     merchCard.removeAttribute('size');
+    merchCard.removeAttribute(ANALYTICS_SECTION_ATTR);
```

Comment:
Thanks, i finally understood that it is here to reflect actual state of the fragment. 
However i find it incomplete to have cleanage being done in bulk for every type of processing, with no specification of what is at stake at the beginning of generic hydrate, and have then processAnalytics that is the one amending that attribute (not specific to analytics, same stands true).
I would have prefer specific code to do everything, probably this part
```
    const { aemFragmentMapping } = merchCard.variantLayout;
    if (!aemFragmentMapping) return;
```
in between basically _implies_ that we need, for each field, 2 places (preprocess & process ?), may be we need subfile / class that has those 2 methods. May be not for this PR :)

cc @yesil